sources/network/tests/network-test.dylan

```diff
@@ -0,0 +1,174 @@
+Module:    network-test
+Synopsis:  Automated test of the network library
+Copyright: (c) 2018 Open Dylan
+           All rights reserved.
+License:   See License.txt in this distribution for details.
+Warranty:  Distributed WITHOUT WARRANTY OF ANY KIND
+
+define test ipv4-address-test ()
+  let addr = make(<internet-address>, name: "www.google.com");
```

Comment:
With regard to https://github.com/dylan-lang/opendylan/pull/1115#discussion_r208739027 this test fails if there is no internet connection (or if Google ceases to exist...) I found it hard to write self-contained tests for networking.

Reply:
Maybe we need a way to mock out DNS lookup. Anyway, that's the least of our worries. I'm gonna merge this.